### ISSUE TITLE
Feature request: resolve local module aliases in M-.

### ISSUE THREAD

**Issue by wilcoxjay** [cpitclaudel/company-coq#116]
Consider the following file:
```
Module N := Nat.
Check N.add.
```
(This code is idiomatic for long module names that have been `Require`d but not `Import`ed.)

Placing the cursor over `N.add` and pressing M-. reports that the definition came from the current buffer. It would be nice if it resolved all the way to Nat.add. I believe the required information is available from a `Locate` call.

**Comment by cpitclaudel** [comment]
Indeed, it looks like Pierre added aliasing information in 2014 (but it's only available in 8.5).
Still, it doesn't cover the 

```
Module M := List.
Locate M.app.
```

@letouzey, do you think it would be easy to extend the commit below to resolve aliases of compatibility notations, so that the example above prints `(alias of List.add)`, or even better `(alias of Datatypes.app)`? (I imagine the former would be easier than the latter)

```
632e7a52e634634d1ba71325b30283dc70ff4b3c
Author:     Pierre Letouzey
AuthorDate: Wed Jul 9 17:28:32 2014 +0200
Locate: also display some information about module aliasing

Typically, if module M has a constant t and module P contains "Include M",
then "Locate t" will now mention that P.t is an alias of M.t
```
